common/terms.html

```diff
@@ -383,6 +383,8 @@
     Its value is either a string (<dfn data-cite="JSON-LD11#dfn-simple-term-definition" data-lt="simple term definition|simple term" data-lt-noDefault>simple term definition</dfn>),
     expanding to an <a>IRI</a>,
     or a map (<a>expanded term definition</a>).
+    <ins cite="#change_api_638"><br/>
+    For <a data-cite="JSON-LD11-API#context-processing-algorithms">context processing</a>, <a>term definition</a> values are converted internally to a dedicated data structure that is easier to process.</ins>
```

Comment:
The problem with this is that that will show up in syntax and framing, as well.

Reply:
We'll need issues for those other specs to have their own version of this update.

Reply:
I created https://github.com/w3c/json-ld-syntax/issues/461 to track this in Syntax, so I think we can probably merge this now.